lib/styleToFilters.js

```diff
@@ -1,6 +1,8 @@
 // import {isExpression} from ;
 let styleSpec = require('@mapbox/mapbox-gl-style-spec');
 
+const DEFAULT_MAX_ZOOM = 22;
```

Comment:
This should likely be 25. Because Mapbox GL, last I checked, supported up to z25. TODO: confirm this.

Reply:
Like mentioned below, the previous behavior of vtshaver is to default to 22 at least (https://github.com/mapbox/vtshaver/blob/master/lib/styleToFilters.js#L32)
Updating it to 25 would cause quite some tests to fail as well.

We should make sure if it should really be 25 or rather stay at 22

Reply:
@springmeyer were you able to confirm this?